OpenRA.Game/Primitives/ActionQueue.cs

```diff
@@ -21,7 +21,7 @@ public class ActionQueue
 	{
 		readonly List<DelayedAction> actions = new List<DelayedAction>();
 
-		public void Add(Action a, int desiredTime)
+		public void Add(Action a, long desiredTime)
```

Comment:
We usually try to avoid using floating-point units in things that affect the world. This is quite heavily used for that. Do we feel that this is safe enough (looks to me like only comparisons and an addition (in `World.cs`) are happening to these numbers.